ckan/setup/ckan_setup.sh

```diff
@@ -25,35 +25,22 @@ do
             pip3 install -e .
             cd $APP_DIR
         fi
-
-        # Point `use` in test.ini to location of `test-core.ini`
-        if [ -f $i/test.ini ];
-        then
-            echo "Updating \`test.ini\` reference to \`test-core.ini\` for plugin $i"
-            ckan config-tool $i/test.ini "use = config:../../src/ckan/test-core.ini"
-        fi
     fi
 done
 
-# Set debug to true
 echo "Enabling debug mode"
 ckan config-tool $CKAN_INI -s DEFAULT "debug = true"
 
+echo "Enabling user creation"
+ckan config-tool $CKAN_INI "ckan.auth.create_user_via_web = true"
+ckan config-tool $CKAN_INI "ckan.auth.create_user_via_api = true"
+
```

Comment:
I don't see any usage of this, any reason why this is important/helpful/useful? Any reason we don't just add this to the `.env` file (just FYI the code block above it inserts something into the `DEFAULT` section, which has to be done with the config-tool and not the `.env` file).

Reply:
You're right.  I thought login wasn't working without it.  But it does.  I just removed this.